app/assets/javascripts/mirador_widget_admin.js

```diff
@@ -37,13 +37,27 @@
       block.find('[data-source-location="' + value + '"]').show();
     }
 
-    function addSelectedItem(block, value) {
-      itemsSection(block).append(itemTemplate(block, value));
+    // TODO: Add some sort of loading animation and clean it up after
+    // complete to help reduce confusion with slow loading manifests
```

Comment:
@ggeisler mind if I split this `TODO` into a Design Needed ticket?

This is about creating some sort of indicator when you click the `Load IIIF item` button that we are actively fetching the manifest (and can be a bit disconcerting when loading a very large and/or slow loading manifest).

Reply:
Sure @jkeck design needed ticket sounds fine and the loading indicator is good idea.